src/frontend/src/pages/AdminToolsPage/BOMConfig/ManufacturerDeleteModal.tsx

```diff
@@ -0,0 +1,62 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Box, IconButton, Typography } from '@mui/material';
+import { useState } from 'react';
+import NERModal from '../../../components/NERModal';
+
+interface ManufacturerDeleteButtonProps {
+  name: string;
+  onDelete: (name: string) => void;
+}
+
+const ManufacturerDeleteButton: React.FC<ManufacturerDeleteButtonProps> = ({
```

Comment:
Lets define the modal here, and the button in the Manufacturer Table. Having the modal be in a component named a button could be potentially confusing in the future

Reply:
Meaning: the button defined in the other file will display the modal defined in this file